Formula/libgnt.rb

```diff
@@ -24,8 +24,18 @@ class Libgnt < Formula
   depends_on "pkg-config" => :build
   depends_on "glib"
 
+  uses_from_macos "libxml2"
+  uses_from_macos "ncurses"
+
   def install
-    ENV["XML_CATALOG_FILES"] = "#{etc}/xml/catalog"
+    ENV["XML_CATALOG_FILES"] = etc/"xml/catalog"
+
+    # Fix ERROR: Problem encountered: ncurses could not be found!
+    # Build only searches for ncurses headers under system prefix '/usr'
```

Comment:
Can we report this upstream?

Reply:
Reported at https://issues.imfreedom.org/issue/LIBGNT-15 and updated comment.